sdk/typescript/src/signers/txn-data-serializers/local-txn-data-serializer.ts

```diff
@@ -336,7 +335,6 @@ export class LocalTxnDataSerializer implements TxnDataSerializer {
     gasObjectId: ObjectId | undefined,
     signerAddress: SuiAddress,
   ): Promise<TransactionData> {
-    // TODO: Allow people to add tip to the reference gas price by using originalTx.data.gasPrice
```

Comment:
Removing since programmable transactions will allow granular control of gas price.